test/clients.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2019 Tekton Authors LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Get access to client objects
+
+To initialize client objects you can use the newClients function. It returns a clients struct
+that contains initialized clients for accessing:
+
+	- Kubernetes resources
+	- Pipelines resources (https://github.com/tektoncd/pipeline)
+	- Triggers resources (https://github.com/tektoncd/triggers)
+
+*/
+
+package test
+
+import (
+	"testing"
+
+	knativetest "github.com/knative/pkg/test"
+	pipelineclientset "github.com/tektoncd/pipeline/pkg/client/clientset/versioned"
+	triggersclientset "github.com/tektoncd/triggers/pkg/client/clientset/versioned"
+	"k8s.io/client-go/kubernetes"
+	// "github.com/tektoncd/triggers/pkg/client/clientset/versioned/typed/triggers/v1alpha1"
```

Comment:
this is a draft so you're probably already gonna do this but we should delete any commented out code before merging 😇